src/view/com/composer/Composer.tsx

```diff
@@ -319,143 +318,140 @@ export const ComposePost = ({
     thread.posts.every(
       post =>
         post.shortenedGraphemeLength <= MAX_GRAPHEME_LENGTH &&
-        !(
-          post.richtext.text.trim().length === 0 &&
-          !post.embed.link &&
-          !post.embed.media &&
-          !post.embed.quote
-        ) &&
+        !isEmptyPost(post) &&
         !(
           post.embed.media?.type === 'video' &&
           post.embed.media.video.status === 'error'
         ),
     )
 
-  const onPressPublish = React.useCallback(
-    async (finishedUploading: boolean) => {
-      if (isPublishing) {
-        return
-      }
-
-      if (!canPost) {
-        return
-      }
+  const onPressPublish = React.useCallback(async () => {
+    if (isPublishing) {
+      return
+    }
 
-      if (
-        !finishedUploading &&
```

Comment:
I believe this wasn't necessary. If the videos are actually uploaded, the condition below will let us through regardless. If not, we shouldn't get through regardless.